chaos-monkey-spring-boot/src/main/java/de/codecentric/spring/boot/chaos/monkey/configuration/toggles/DefaultChaosToggleNameMapper.java

```diff
@@ -0,0 +1,21 @@
+package de.codecentric.spring.boot.chaos.monkey.configuration.toggles;
+
+import de.codecentric.spring.boot.chaos.monkey.component.ToggleType;
+import de.codecentric.spring.boot.chaos.monkey.configuration.ChaosMonkeyProperties;
+
+public class DefaultChaosToggleNameMapper implements ChaosToggleNameMapper {
+  protected final ChaosMonkeyProperties chaosMonkeyProperties;
+
+  public DefaultChaosToggleNameMapper(ChaosMonkeyProperties chaosMonkeyProperties) {
```

Comment:
I think its sufficient if we pass togglePrefix to the class as the other properties arent really necessary for mapping the name